Sources/Geohash/Geohash.swift

```diff
@@ -102,17 +102,17 @@ public struct Geohash {
 
 public extension Geohash {
     enum Precision: Int {
-        case twentyFiveHundredKilometers = 1    // ±2500 km
-        case sixHundredThirtyKilometers         // ±630 km
-        case seventyEightKilometers             // ±78 km
-        case twentyKilometers                   // ±20 km
-        case twentyFourHundredMeters            // ±2.4 km
-        case sixHundredTenMeters                // ±0.61 km
-        case seventySixMeters                   // ±0.076 km
-        case nineteenMeters                     // ±0.019 km
-        case twoHundredFourtyCentimeters        // ±0.0024 km
-        case sixtyCentimeters                   // ±0.00060 km
-        case seventyFourMillimeters             // ±0.000074 km
+        case _1_2500km = 1  // ±2500 km
+        case _2_630km       // ±630 km
+        case _3_78km        // ±78 km
+        case _4_20km        // ±20 km
+        case _5_2400m       // ±2.4 km
+        case _6_610m        // ±0.61 km
+        case _7_76m         // ±0.076 km
+        case _8_19m         // ±0.019 km
+        case _9_2400cm      // ±0.0024 km
+        case _10_60cm       // ±0.00060 km
+        case _11_74mm       // ±0.000074 km
```

Comment:
What about something like this?
```
enum Precision: Int {
     case km2500 = 1    // ±2500 km
     case km630         // ±630 km
     case km78          // ±78 km
     case km20          // ±20 km
     case m2400         // ±2.4 km
     case m610          // ±0.61 km
     case m76           // ±0.076 km
     case m19           // ±0.019 km
     case cm240         // ±0.0024 km
     case cm60          // ±0.00060 km
     case mm74          // ±0.000074 km
}
```

Reply:
Hmm, I don't feel it's that better.  "km" and "cm" look ok but something like `m610` doesn't look a distance to me.
```swift
Geohash.encode(latitude: lat, longitude: lon, precision: .twoHundredFourtyCentimeters) 
Geohash.encode(latitude: lat, longitude: lon, precision: ._240cm) 
Geohash.encode(latitude: lat, longitude: lon, precision: .cm240) 
```
While I'm super thrilled to use `_` prefix, I'd prefer 2nd one much better to 3rd one.